lib/content-api.js

```diff
@@ -60,6 +73,21 @@ function transformData(obj, dirpath) {
 }
 
 ContentApi.prototype.getRecord = function getRecord(slug) {
+  return new Promise((resolve) => {
+    this.getRecordFromApi(slug)
+      .then((record) => {
+        logger.verbose(`Loaded from content API: ${slug}`);
+        resolve(record);
+      })
+      .catch((error) => {
+        logger.error(error);
```

Comment:
Well, we won't have the Content Store in prod atm so I wouldn't log the error. We should though as soon as we have the two pieces connected and live.